File: c++/palindrome.cpp
#include <iostream>
#include<bits/stdc++.h>
using namespace std;
int main() {
    int n,rev=0,r,temp;
    cout<<"Enter a number: "<<endl;
    cin>>n;
    temp=n;
    while(n>0)
    {
        r=n%10;
        rev=rev*10+r;
        n=n/10;
    }
    cout<<"Reverse "<<" is"<<endl<<rev<<endl;
    
    if(temp==rev)
    {
        cout<<temp<< " is palindrome";
    }
    else
    {
        cout<<temp<<" is not palindrome";
    }
    return 0;
}

File: c#/palindrome.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        Console.WriteLine ("Enter a number: ");
        int n=Convert.ToInt32(Console.ReadLine());
        int x=n;
        int r,rev=0;
        
        while(n>0)
        {
            r=n%10;
            rev=rev*10+r;
            n=n/10;
        }
        Console.WriteLine("Reverese "+rev);
        
        if(rev==x)
        {
            Console.WriteLine("Palindrome no");
        }
        else
        {
            Console.WriteLine("Non Palindrome no");
        }
    }
}
